static/app/components/confirm.tsx

```diff
@@ -129,7 +133,7 @@ type Props = OpenConfirmOptions & {
  * Opens a confirmation modal when called. The procedural version of the
  * `Confirm` component
  */
-export const openConfirmModal = ({
+const openConfirmModal = ({
```

Comment:
this function was not being imported by other components

Reply:
Please leave this exported. I am going to use it in my _admin rebuild.